### PR TITLE
update setup key edit layout

### PR DESCRIPTION
**Pull request by saroojbkhari**

### CODE REVIEW

File: src/components/SetupKeyEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+          key={0}
+        >
+          <Button onClick={onCancel}>Cancel</Button>
+          <Button
+            type="primary"
+            style={{
+              height: "100%",
+              fontSize: "14px",
+              borderRadius: "2px",
+            }}
```
Comment (braginini) [inline src/components/SetupKeyEdit.tsx:484]:
```suggestion
```

Comment (braginini) [reply src/components/SetupKeyEdit.tsx:484]: no need for these styles, let's do similar to other pages (see the UserEdit component)

**Review by braginini — CHANGES_REQUESTED** [review]
Additionally to requested changes:
1. There is a bug when creating a new key and setting the reusable switch to true but not changing the available uses. When you click Create key, the API endpoind is called with 0 which means unlimited. We should set it to 0 by default (instead of current 1)  
(The bug was there before the PR)
2. I think that we should do the same layout as in the UserEdit component:

Screenshot from 2023-05-31 16-17-26

File: src/components/SetupKeyEdit.tsx [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+              >
+                <Paragraph
+                  style={{
+                    whiteSpace: "pre-line",
+                    margin: 0,
+                    fontWeight: "bold",
+                  }}
+                ></Paragraph>
+                {formSetupKey.type === "one-off" ? "One-off" : "Reusable"},
+                Available uses
```
[the remark this answers is not in the document]
Comment (saroojbkhari) [reply src/components/SetupKeyEdit.tsx:383]: done

File: src/components/SetupKeyEdit.tsx (hunk 1 above)
Comment (saroojbkhari) [reply src/components/SetupKeyEdit.tsx:484]: done

**Pull request merged**